doc/api-examples/c++/list_nodes.cpp

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+#include <dbus/dbus.h>
+#include <iostream>
+
+void handle_structure(DBusMessageIter *structIter) {
```

Comment:
Nit: Maybe `print_node`? Its only handling the specific structure returned by list nodes `(sos)`, nothing generic.